packages/pluggableWidgets/image-viewer-web/package.json

```diff
@@ -28,11 +28,14 @@
   },
   "devDependencies": {
     "@mendix/pluggable-widgets-tools": ">=8.9.2",
+    "@rollup/plugin-commonjs": "^16.0.0",
```

Comment:
This plugin is already being delivered in the piw-tools, so its not necessary to re-add